### DISCUSSION TITLE
Netbox Binary image

### DISCUSSION THREAD

**Discussion by berkeleycowboy** [netbox-community/netbox#14834] [Help Wanted!]
I've been trying to get Netbox running on Redhat version 8 for approximately a year.   For whatever reason Secure LDAP does not work.  Secure LDAP is a requirement in my environment.  

I have heard rumor that a Netbox Binary images (kind of like an appliance) can be downloaded.  

Can someone direct me to where the Binary images are stored ?

**Comment by DanSheps** [comment]
You should stick to the original thread.
